ui/raidboss/data/02-arr/raid/t1.ts

```diff
@@ -101,7 +108,7 @@ export default {
       netRegexKo: NetRegexes.message({ line: '알라그 유적 will be sealed off.*?', capture: false }),
       delaySeconds: 35,
       suppressSeconds: 5,
-      infoText: (_data, _matches, output) => output.text(),
+      infoText: (_data, _matches, output) => output.text?.(),
```

Comment:
I think we should disable this null check for trigger files.

Reply:
If we could disable the undefined check just for this one object, I think that would be good, but I don't want to do it in general.  I think the benefit of the undefined check is better than the irritating at having to write this.  The only other alternative is `output.text!()`, which isn't much better.

I tried a lot of different things, but I don't think there's any way to say that `output.text` is always defined.  Unfortunately, `Proxy` does not seem like a first-class TypeScript object, and so it doesn't know that it can look at the `get` function to understand what types can be returned.

Reply:
Looks like currently in TypeScript `Proxy` just return a type as same as it's target

Reply:
Unfortunately, if you make `Output` be `any`, then any call on that `any` value like `text()` is also an "unsafe call on an any value" error.  So, I'm not sure that's the solution.